### DISCUSSION TITLE
architechair/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#50] [Announcements]
# architechair/

What Am I Even Doing

https://www.supergoodcode.com/architechair/

**Comment by FireBurn** [comment]
It was a wee bit painful, but big kudos for getting it done. I look forward to seeing what you do next and hopefully some improvements to the CI

**Comment by Scrumplex** [comment]
3 🤨

**Comment by ChaosFenix** [comment]
Oh come on. We all know everyone who works for valve forgets how to count to 3. We will forever have CS2 and we will get a steam deck 2 only to be followed by nothing in perpetuity.

**Reply by wertig0n** [reply]
Oh ye of little faith...

It is obvious Valve is working with release numbers using the exponential function -3^(1-x) + 3. If you graph it, you will see that no matter how many years (x) it will never reach 3.

However, expect Half Life e at *some* point. Also Half Life i is a possibility.
